### PR TITLE
Update side pane heading styles and correct video effects side pane width

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What

- Increase font size and padding of the desktop side pane header
- Update close button padding to match
- Decrease the side pane width for the video effects

# Why

Match new figmas

# How Tested
Locally - see updated snapshots to see visual changes

### CODE REVIEW

**Review by dmceachernmsft — COMMENTED** [review]
image
looks like there is some alignment issues here

File: packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx [hunk 1]
```diff
@@ -347,6 +349,8 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
             </Stack.Item>
             <SidePane
               mobileView={props.mobileView}
+              /* @conditional-compile-remove(video-background-effects) */
+              maxWidth={isVideoPaneOpen ? '19.5rem' : undefined}
```
Comment (prabhjot-msft) [inline packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx:353]: Does the maxwidth number `19.5rem` is provided by design or we are maitaining max height from our side, just curious :)

**Review by prabhjot-msft — COMMENTED** [review]
I have noticed a slight difference in margin from designs. In our current design we have lesser margin and also AlexP pointed in one of the bug bashes.
Screenshot 2023-06-14 at 1 48 01 PM
bug-margin
Bug: https://skype.visualstudio.com/SPOOL/_workitems/edit/3280992

As of current implementation, looks like header is shifted along with the content. Maybe we can check with AlexP
Screenshot 2023-06-14 at 1 49 15 PM

File: packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx (hunk 1 above)
Comment (JamesBurnside) [reply packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx]: Yeah, should've been 17.5 though my bad! Updated

File: packages/react-composites/tests/browser/snapshots/beta/tests/browser/call/hermetic/VideoEffects.test.ts-snapshots/custom-video-effects-config-screen-pane-none-selected-Desktop-Chrome-linux.png
[the remark this answers is not in the document]
Comment (JamesBurnside) [reply packages/react-composites/tests/browser/snapshots/beta/tests/browser/call/hermetic/VideoEffects.test.ts-snapshots/custom-video-effects-config-screen-pane-none-selected-Desktop-Chrome-linux.png:1]: Yeah figma has a tiny offset:
image

**Review by dmceachernmsft — APPROVED** [review]

**Comment by JamesBurnside** [comment]
> I have noticed a slight difference in margin from designs. In our current design we have lesser margin and also AlexP pointed in one of the bug bashes.
> As of current implementation, looks like header is shifted along with the content. Maybe we can check with AlexP 

Updated to have the tighter margins

**Review by prabhjot-msft — APPROVED** [review]

**Pull request merged**